modules/rosa-hcp-cluster-terraform-file-creation.adoc

```diff
@@ -0,0 +1,278 @@
+  cluster_name           = local.cluster_name
+  openshift_version      = var.openshift_version
+  account_role_prefix    = local.cluster_name
+  operator_role_prefix   = local.cluster_name
+  replicas               = local.worker_node_replicas
+  aws_availability_zones = local.region_azs
+  create_oidc            = true
+  private                = var.private_cluster
+  aws_subnet_ids         = var.create_vpc ? var.private_cluster ? module.vpc[0].private_subnets : concat(module.vpc[0].public_subnets, module.vpc[0].private_subnets) : var.aws_subnet_ids
+  create_account_roles   = true
+  create_operator_roles  = true
+}
+EOF
+----
+
+. Create the `variables.tf` file by running the following command:
++
+[NOTE]
+====
+Copy and edit this file _before_ running the command to build your cluster.
+====
++
+[source,terminal]
+----
+$ cat<<-EOF>variables.tf
+variable "openshift_version" {
+  type        = string
+  default     = "4.14.20"
+  description = "Desired version of OpenShift for the cluster, for example '4.1.0'. If version is greater than the currently running version, an upgrade will be scheduled."
+}
+
+variable "create_vpc" {
+  type        = bool
+  description = "If you would like to create a new VPC, set this value to 'true.' If you do not want to create a new VPC, set this value to 'false.'"
+}
+
+# ROSA Cluster info
+variable "cluster_name" {
+  default     = null
+  type        = string
+  description = "The name of the ROSA cluster to create"
+}
+
+variable "additional_tags" {
+  default = {
+    Terraform   = "true"
+    Environment = "dev"
+    TFOwner     = "mobb@redhat.com"
+  }
+  description = "Additional AWS resource tags"
+  type        = map(string)
+}
+
+variable "multi_az" {
+  type        = bool
+  description = "Multi AZ Cluster for High Availability"
+  default     = true
+}
+
+variable "path" {
+  description = "(Optional) The arn path for the account/operator roles as well as their policies."
+  type        = string
+  default     = null
+}
+
+variable "machine_type" {
+  description = "The AWS instance type used for your default worker pool"
+  type        = string
+  default     = "m5.xlarge"
+}
+
+variable "worker_node_replicas" {
+  default     = 3
+  description = "Number of worker nodes to provision. Single zone clusters need at least 2 nodes, multizone clusters need at least 3 nodes"
+  type        = number
+}
+
+
+variable "autoscaling_enabled" {
+  description = "Enables autoscaling. This variable requires you to set a maximum and minimum replicas range using the 'max_replicas' and 'min_replicas' variables."
+  type        = string
+  default     = "false"
+}
+
+variable "min_replicas" {
+  description = "The minimum number of replicas for autoscaling."
+  type        = number
+  default     = 3
+}
+
+variable "max_replicas" {
+  description = "The maximum number of replicas not exceeded by the autoscaling functionality."
+  type        = number
+  default     = 3
+}
+
+variable "proxy" {
```

Comment:
As we're not passing this variable to the module, it's not used and can be removed as this is a quick start guide